MapboxNavigation/StepsViewController.swift

```diff
@@ -196,35 +196,30 @@ extension StepsViewController: UITableViewDataSource {
     
     func updateCell(_ cell: StepTableViewCell, at indexPath: IndexPath) {
         let step = sections[indexPath.section][indexPath.row]
-        
-        cell.instructionsView.maneuverView.step = step
        
         let usePreviousLeg = indexPath.section != 0 && indexPath.row == 0
-        let leg = routeProgress.route.legs[indexPath.section]
-        let arrivalSecondaryInstruction = leg.destination.name
         
         if usePreviousLeg {
```

Comment:
@JThramer did a nice refactor of this if statement in https://github.com/mapbox/mapbox-navigation-ios/pull/1063/files#diff-05dcf05089a2947433b80faae897f68dR207. Wonder if we could reuse this.